hubstorage/client.py

```diff
@@ -40,7 +40,7 @@ def _hc_retry_on_exception(err):
 class HubstorageClient(object):
 
     DEFAULT_ENDPOINT = 'https://storage.scrapinghub.com/'
-    USERAGENT = 'python-hubstorage/{0}'.format(__version__)
+    DEFAULT_USER_AGENT = 'python-hubstorage/{0}'.format(__version__)
```

Comment:
You can keep `DEFAULT_USER_AGENT`, but object should as well contain `USERAGENT` attribute - e.g. as property that returns `DEFAULT_USER_AGENT` with deprecation warning and appropriate setter